components/Collections/src/MDCCollectionViewController.m

```diff
@@ -434,12 +434,12 @@ - (BOOL)collectionView:(UICollectionView *)collectionView
 
 - (void)collectionView:(UICollectionView *)collectionView
```

Comment:
Should we explicitly state that if you subclass MDCCollectionViewController, you should call super on the delegate method?